src/Lister/SimpleSubPageFinder.php

```diff
@@ -201,6 +219,11 @@ private function getOptions() {
 
 		$options['LIMIT'] = (int)$this->options[self::OPT_LIMIT];
 
+		$sort_oder = $this->options[self::OPT_SORT_ORDER];
+		if ($sort_oder != self::NONE) {
+            $options['ORDER BY'] = "LENGTH(page_title) $sort_oder, page_title $sort_oder";
```

Comment:
Is there no way to escape these values? My knowledge of the MW DB stuff is a bit rusty... Having no explicit escaping here makes me uneasy.